rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -3,10 +3,11 @@
 %global foreman_proxy_bundlerd_dir %{_datadir}/foreman-proxy/bundler.d
 %global foreman_proxy_pluginconf_dir %{_sysconfdir}/foreman-proxy/settings.d
 %global spool_dir %{_var}/spool/foreman-proxy/openscap
+%global content_dir %{_var}/lib/openscap
```

Comment:
Sounds like this should be under /var/lib/foreman-proxy then, if it's files managed by the foreman-proxy process.